<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>839</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>839</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionCircle"/>
   <addaction name="actionTriangle"/>
   <addaction name="actionTrapezium"/>
   <addaction name="actionLine"/>
   <addaction name="actionText"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionRestore"/>
   <addaction name="separator"/>
   <addaction name="actionRotateLeft"/>
   <addaction name="actionRotateRight"/>
   <addaction name="actionFrontPlace"/>
   <addaction name="actionBackPlace"/>
   <addaction name="actionCombine"/>
   <addaction name="actionScattered"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/zoomin.png</normaloff>:/images/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/rotateleft.png</normaloff>:/images/rotateleft.png</iconset>
   </property>
   <property name="text">
    <string>左旋转</string>
   </property>
  </action>
  <action name="actionRotateRight">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/rotateright.png</normaloff>:/images/rotateright.png</iconset>
   </property>
   <property name="text">
    <string>右旋转</string>
   </property>
   <property name="toolTip">
    <string>右旋转</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/zoomout.png</normaloff>:/images/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/420.bmp</normaloff>:/images/420.bmp</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
  </action>
  <action name="actionFrontPlace">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/528.bmp</normaloff>:/images/528.bmp</iconset>
   </property>
   <property name="text">
    <string>前置</string>
   </property>
   <property name="toolTip">
    <string>前置</string>
   </property>
  </action>
  <action name="actionBackPlace">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/526.bmp</normaloff>:/images/526.bmp</iconset>
   </property>
   <property name="text">
    <string>后置</string>
   </property>
   <property name="toolTip">
    <string>后置</string>
   </property>
  </action>
  <action name="actionCombine">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/UNGROUP.BMP</normaloff>:/images/UNGROUP.BMP</iconset>
   </property>
   <property name="text">
    <string>组合</string>
   </property>
   <property name="toolTip">
    <string>组合</string>
   </property>
  </action>
  <action name="actionScattered">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/128.bmp</normaloff>:/images/128.bmp</iconset>
   </property>
   <property name="text">
    <string>打散</string>
   </property>
   <property name="toolTip">
    <string>打散</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/108.bmp</normaloff>:/images/108.bmp</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/132.bmp</normaloff>:/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/RECTANGL.BMP</normaloff>:/images/RECTANGL.BMP</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
   <property name="toolTip">
    <string>矩形</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/ELLIPSE.BMP</normaloff>:/images/ELLIPSE.BMP</iconset>
   </property>
   <property name="text">
    <string>椭圆</string>
   </property>
   <property name="toolTip">
    <string>椭圆</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/08.JPG</normaloff>:/images/08.JPG</iconset>
   </property>
   <property name="text">
    <string>圆形</string>
   </property>
   <property name="toolTip">
    <string>圆形</string>
   </property>
  </action>
  <action name="actionTriangle">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/Icon1242.ico</normaloff>:/images/Icon1242.ico</iconset>
   </property>
   <property name="text">
    <string>三角形</string>
   </property>
   <property name="toolTip">
    <string>三角形</string>
   </property>
  </action>
  <action name="actionTrapezium">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/FREEFORM.BMP</normaloff>:/images/FREEFORM.BMP</iconset>
   </property>
   <property name="text">
    <string>梯形</string>
   </property>
   <property name="toolTip">
    <string>梯形</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/LINE.BMP</normaloff>:/images/LINE.BMP</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
   <property name="toolTip">
    <string>直线</string>
   </property>
  </action>
  <action name="actionText">
   <property name="icon">
    <iconset resource="drawresource.qrc">
     <normaloff>:/images/800.bmp</normaloff>:/images/800.bmp</iconset>
   </property>
   <property name="text">
    <string>文字</string>
   </property>
   <property name="toolTip">
    <string>文字</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qwgraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="drawresource.qrc"/>
 </resources>
 <connections/>
</ui>
